src/renderer/config.js

```diff
@@ -1,5 +1,7 @@
 import path from 'path'
-
+export const isInElectron = window && window.process && window.process.type === 'renderer'
```

Comment:
The renderer is always executed in Electron, so you can use `process.platform` like `isLinux`.